docs/pages/enroll-resources/application-access/guides/vnet.mdx

```diff
@@ -100,28 +95,17 @@ Each cluster has a configurable IPv4 CIDR range which VNet uses when assigning I
 applications from that cluster. Root and leaf clusters can use different ranges. The default is
 `100.64.0.0/10` and it can be changed by setting the `ipv4_cidr_range` field of the VNet config.
 
-Create a file called `vnet-config.yaml`:
-
-```yaml
-kind: vnet_config
-version: v1
-metadata:
-  name: vnet-config
-spec:
-  ipv4_cidr_range: "100.64.0.0/10"
-```
-
-Create the VNet config:
+Edit the VNet config:
 
 ```code
-$ tctl create vnet-config.yaml
-vnet_config has been created
+$ tctl edit vnet_config
 ```
 
-If the config already exists, you can use `tctl edit` instead:
+Edit the `spec` section of the config to include the `ipv4_cidr_range` field:
 
-```
-$ tctl edit vnet_config
+```yaml
+spec:
+  ipv4_cidr_range: "100.64.0.0/10"
```

Comment:
I specifically included just the relevant sections. This is because `tctl edit` includes `metadata.revision` and the edit is going to fail if this field doesn't match the current state of the backend or if it's removed completely. If I included the whole resource with or without `metadata.revision` and the user just pasted that, it wouldn't work. :(

This behavior makes `tctl edit` a bit bad for docs, but it's so much more convenient than having to create a separate yaml file and then execute `tctl create`.

Reply:
The alternative is to make this code block `diff` syntax instead of `yaml`, then you could highlight the actual changes by prefixing those lines with `+`.

Reply:
Ah yes, I see it being used in other places in the docs, for example https://goteleport.com/docs/admin-guides/access-controls/device-trust/enforcing-device-trust/#web-application-support

@zmb3 Could you approve this PR?